app/controllers/dashboard/protocol_filters_controller.rb

```diff
@@ -32,7 +32,16 @@ def create
       flash[:alert] = 'Search Failed to Save.'
     end
 
-    @protocol_filters = ProtocolFilter.latest_for_user(@user.id, 5)
+    @protocol_filters = ProtocolFilter.latest_for_user(@user.id, 15)
```

Comment:
We should define "15" as a static value on the ProtocolFilter model since it's used in more than one place.